content/en/blog/_posts/2023-03-08-Updates-to-the-Auto-refreshing-Official-CVE-Feed/index.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: blog
+title: Updates to the Auto-refreshing Official CVE Feed
+date: 2023-03-22
+slug: k8s-cve-feed-beta
+---
+
+**Authors**: Cailyn Edwards (Shopify), Pushkar Joglekar (Credit Karma), Mahé Tardy (Isovalent)
+
+Since launching the [Auto-refreshing Official CVE feed](/docs/reference/issues-security/official-cve-feed/) as an alpha
+feature in the 1.25 release, we have made significant improvements and updates. We are excited to announce the release of the
+beta version of the feed. This blog post will outline the feedback received, the changes made, and talk about what is
+planned for the stable release.
```

Comment:
```suggestion
feature in the 1.25 release, we have made significant improvements and updates. We are excited to announce the release of the
`beta` version of the feed. This blog post will outline the feedback received, the changes made, and talk about how you can help as we prepare to make this a `stable` feature in a future Kubernetes Release
```

Reply:
Will wait for a decision on the `highlight` and then address